src/InteractiveMars/Mars.js

```diff
@@ -0,0 +1,62 @@
+/* TO-DO 
+- Make marker onClick zoom less janky (or disable alltogether)
+- Label markers with rover names without having to hover? (ThreeJS text geometry?)
+- Make enter button look pretty
+*/
+
+import React, { useState } from "react";
+import ReactGlobe from "react-globe";
+import { Markers } from "./Markers";
+import "tippy.js/dist/tippy.css";
+import "tippy.js/animations/scale.css";
+
+//Function to render onHover tooltips
+function markerTooltipRenderer(marker) {
+  return `ROVER: ${marker.rover} `;
+}
+
+//Options for globe, more options found at https://react-globe.netlify.app/usage/options
+const options = {
+  markerTooltipRenderer,
+  cameraAutoRotateSpeed: 0.8,
+  cameraMaxPolarAngle: (Math.PI * 9) / 16,
+  cameraMinPolarAngle: (Math.PI * 7) / 16,
+  cameraRotateSpeed: 0.5,
+  enableCameraAutoRotate: true,
+  enableCameraRotate: true,
+  enableCameraZoom: false
+};
+
+//Globe export
+
+export function Mars() {
+  // Get marker data from Markers.js
+  const [markers] = useState(Markers);
+
+  return (
+    <div>
+      {/* Enter mars button */}
+      <form style={{
+        position: 'absolute', left: '50%', top: '50%',
+        transform: 'translate(-50%, -50%)'
+      }}
+        action="http://localhost:3000/rovers" method="get">
+        <button>ENTER MARS</button>
+      </form>
+
+      {/* Globe and globe props, refer to documentation @ https://react-globe.netlify.app/*/ }
+      <ReactGlobe
+        //focus={null} should prevent globe zoom on marker click but it doesn't :(
+        focus={null}
+        //Make sure height & width props are not percentage based, else globe will infinitely resize
+        height="100vh"
+        //Mars texture, maybe download and keep part of files incase site unreachable at any point?
+        globeTexture="https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/f/9b7029c3-9717-4658-9066-11c30aa24029/dcsauye-ba810e63-20e3-4ae9-a73c-9201ed87e67d.png/v1/fill/w_1280,h_640,q_80,strp/mars_texture_map__rare_version__by_oleg_pluton_dcsauye-fullview.jpg?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ1cm46YXBwOiIsImlzcyI6InVybjphcHA6Iiwib2JqIjpbW3siaGVpZ2h0IjoiPD02NDAiLCJwYXRoIjoiXC9mXC85YjcwMjljMy05NzE3LTQ2NTgtOTA2Ni0xMWMzMGFhMjQwMjlcL2Rjc2F1eWUtYmE4MTBlNjMtMjBlMy00YWU5LWE3M2MtOTIwMWVkODdlNjdkLnBuZyIsIndpZHRoIjoiPD0xMjgwIn1dXSwiYXVkIjpbInVybjpzZXJ2aWNlOmltYWdlLm9wZXJhdGlvbnMiXX0.X7rDRGm_uGFDxKyzB7nkfQuSxXmT87Hhg7CsW79lJTI"
```

Comment:
Can we add image to our source code?
(and then change the link to point at our copy)

Reply:
Doesn't seem to like this, had an issue initially where it would only accept images from certain URLs. The author of the library says something about the endpoint requiring cross-origin, but I should think that only effects it if you're passing in a URL

Reply:
hmm that's odd - does it make an API request to load things? (if not we should be fine with cross origin)

> Doesn't seem to like this

What did you try? What happened?